docs/conf.py

```diff
@@ -125,9 +125,9 @@
 # adapted from: http://rackerlabs.github.io/docs-rackspace/tools/rtd-tables.html
 # and https://github.com/altair-viz/altair/pull/418/files
 # https://github.com/rtfd/sphinx_rtd_theme/issues/117
-def setup(app):
+# def setup(app):
     # Add our custom CSS overrides
-    app.add_stylesheet('theme_overrides.css')
+  #  app.add_stylesheet('theme_overrides.css')
```

Comment:
Is the setup function no longer necessary?

If so, we should delete it and the comment above. However, it's not clear to me why the change was made?